modules/react/menu/lib/MenuCard.tsx

```diff
@@ -50,21 +52,24 @@ export const MenuCard = createSubcomponent('div')({
   displayName: 'Menu.Card',
   modelHook: useMenuModel,
   elemPropsHook: useMenuCard,
-})<MenuCardProps>((elemProps, Element, model) => {
+})<MenuCardProps>(({minWidth = px2rem(1), ...elemProps}, Element, model) => {
```

Comment:
```suggestion
})<MenuCardProps>(({minWidth, ...elemProps}, Element, model) => {
```
you set default in stencil